x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -978,7 +979,128 @@ describe('Task Runner', () => {
     }
   );
 
-  test('actionsPlugin.execute is not called when notifyWhen=onActionGroupChange and alert alert state does not change', async () => {
+  test.each(ephemeralTestParams)(
+    'skips firing actions for active alert if alert is throttled %s',
+    async (nameExtension, customTaskRunnerFactoryInitializerParams, enqueueFunction) => {
+      (
+        customTaskRunnerFactoryInitializerParams as TaskRunnerFactoryInitializerParamsType
+      ).actionsPlugin.isActionTypeEnabled.mockReturnValue(true);
+      customTaskRunnerFactoryInitializerParams.actionsPlugin.isActionExecutable.mockReturnValue(
+        true
+      );
+      actionsClient.ephemeralEnqueuedExecution.mockResolvedValue(mockRunNowResponse);
+      ruleType.executor.mockImplementation(
+        async ({
+          services: executorServices,
+        }: AlertExecutorOptions<
+          AlertTypeParams,
+          AlertTypeState,
+          AlertInstanceState,
+          AlertInstanceContext,
+          string
+        >) => {
+          executorServices.alertInstanceFactory('1').scheduleActions('default');
```

Comment:
Will have to update this since [this PR](https://github.com/elastic/kibana/pull/124390) was merged